extension-api/app/core/subscription.py

```
@@ -276,56 +299,80 @@ async def check_collection_limit(user_id: str) -> Tuple[bool, str, Optional[str]
         """
         Check if user can add more links to a collection.
         Returns (can_add, error_message, plan_slug)
         """
-        logger.info(f"🔍 CHECK_LINKS_LIMIT - Called for user: {user_id}, collection: {collection_id}, links_to_add: {links_to_add}")
-        
+        logger.info(
+            f"🔍 CHECK_LINKS_LIMIT - Called for user: {user_id}, collection: {collection_id}, links_to_add: {links_to_add}"
+        )
+
         try:
             logger.info(f"🔍 Getting user subscription...")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)